docs/source/examples/classification_and_regression.rst

```diff
@@ -0,0 +1,312 @@
+
+.. code-block:: python
+
+    network = ClassifierLocalizer(model_name='resnet18')
+
+    def freeze_weights(network):
+        for name, param in network.named_parameters():
+            if not name.startswith('fc.'):
+                param.requires_grad = False
+
+    freeze_weights(network)
+    print(network)
+
+Loss function
+=============
+
+As we discussed before, we have two different tasks in this example. These tasks need different loss functions; Cross-Entropy loss for the classification and Mean Square Error loss for the regression. Below, we define a new loss function class that sums both losses to considers them simultaneously. However, as the regression is relatively a harder task, we apply a higher weight to MSEloss.
+
+.. code-block:: python
+
+    class ClassificationRegressionLoss(nn.Module):
+        def __init__(self, W):
+            super(ClassificationRegressionLoss, self).__init__()
+            self.ce_loss = nn.CrossEntropyLoss() # size_average=False
+            self.mse_loss = nn.MSELoss()
+            self.W = W
+            
+        def forward(self, y_pred, y_true):
+            loss_cls = self.ce_loss(y_pred[0], Variable(y_true[0][:,gender_index])) # Cross Entropy Error (for classification)
+            loss_reg1 = self.mse_loss(y_pred[1][:,0], y_true[1][:,0]/h) # Mean Squared Error for X_L
+            loss_reg2 = self.mse_loss(y_pred[1][:,1], y_true[1][:,1]/w) # Mean Squared Error for Y_L
+            loss_reg3 = self.mse_loss(y_pred[1][:,2], y_true[1][:,2]/h) # Mean Squared Error for X_R
+            loss_reg4 = self.mse_loss(y_pred[1][:,3], y_true[1][:,3]/w) # Mean Squared Error for Y_R
+            total_loss = loss_cls + self.W * (loss_reg1 + loss_reg2 + loss_reg3 + loss_reg4)
+            return total_loss
+
+Training
+========
+
+.. code-block:: python
+
+    optimizer = optim.Adam(network.parameters(), lr=learning_rate, weight_decay=0)
+    loss_function = ClassificationRegressionLoss()
+    #Step_Learning_Rate = StepLR(step_size=2 , gamma=0.1, last_epoch=-1, verbose=False)
+    exp = Experiment('./two_task_example', network, optimizer=optimizer, loss_function=loss_function, device="all")
+    exp.train(train_dataloader, valid_dataloader, callbacks=callbacks, epochs=num_epochs)
+
+Evaluation
+==========
+
+As you have also noticed from the training logs, in this try we achieved the best performance (considering the validation loss) at the 15th epoch. The weights of the network for the corresponding epoch have been automatically saved by the `Experiment` function and we use these parameters to evaluate our algorithm visually. For this purpose, we utulize the load_checkpoint method and set its argument to `best` to load the best weights of the model automatically. Finally,  we take advantage of the `evaluate` function of Poutyne, and apply it to the validation dataset. It provides us the predictions as well as the ground-truth for comparison, in case of need.
```

Comment:
```suggestion
As you have also noticed from the training logs, in this try we achieved the best performance (considering the validation loss) at the 15th epoch. The weights of the network for the corresponding epoch have been automatically saved by the `Experiment` function and we use these parameters to evaluate our algorithm visually. For this purpose, we utilize the `load_checkpoint` method and set its argument to `best` to load the best weights of the model automatically. Finally,  we take advantage of the `evaluate` function of Poutyne, and apply it to the validation dataset. It provides us the predictions as well as the ground-truth for comparison, in case of need.
```